src/main/resources/dbchangelogs/changelog-4.15.0-delete-department-calendar-on-department-deletion.xml

```diff
@@ -7,7 +7,7 @@
 
     <preConditions>
       <tableExists tableName="department_calendar"/>
-      <foreignKeyConstraintExists foreignKeyName="fk_department_calendar_department"/>
+      <foreignKeyConstraintExists foreignKeyTableName="department_calendar" foreignKeyName="fk_department_calendar_department"/>
```

Comment:
bekommen wir hier ein problem mit den checksums?

Reply:
ne, hatte das auch im Commit mal beschrieben

This does not affect the md5 hash see
https://forum.liquibase.org/t/preconditions-affect-checksum/959